ru/mail/chrome/messenger/AccountWizard.dtd

```diff
@@ -66,55 +66,55 @@
      these variables. Also, translate haveSmtp1.prefix and haveSmtp1.suffix as a single sentence, inserting
      text after the "&quot;" entity in haveSmtp1.suffix, if required grammatically.
 -->
-<!ENTITY haveSmtp1.prefix "Будет использоваться уже существующий сервер исходящей почты (SMTP) — «">
-<!ENTITY haveSmtp1.suffix "&quot;, will be used. You can modify outgoing server settings by choosing Account Settings from the Tools menu.">
+<!ENTITY haveSmtp1.prefix "Будет использоваться существующий сервер исходящей почты (SMTP) «">
+<!ENTITY haveSmtp1.suffix "». Чтобы изменить параметры сервера исходящей почты, в меню «Инструменты» выберите «Параметры учетной записи».">
 
 <!-- Entities for Login page -->
 
-<!ENTITY loginTitle.label "Имена пользователей">
-<!ENTITY usernameDesc.label "Введите имя пользователя, данное вам провайдером электронной почты">
+<!ENTITY loginTitle.label "Имена пользователя">
+<!ENTITY usernameDesc.label "Введите имя пользователя сервера входящей почты, полученное от провайдера электронной почты">
 <!-- LOCALIZATION NOTE (usernameExample.label) : do not translate "&quot;jsmith&quot;" in below line -->
-<!ENTITY usernameExample.label "(например, «i_ivanov»).">
-<!ENTITY usernameLabel.label "Имя пользователя:">
+<!ENTITY usernameExample.label "(например, i_ivanov).">
```

Comment:
Yes, I saw this. Will implement or comment all corrections tomorrow.
Yury

From: Sherman Dickman 
Sent: Sunday, August 28, 2016 7:59 PM
To: postbox/localizations 
Cc: yshchebet ; Mention 
Subject: Re: [postbox/localizations] RU translation Aug 22 (#28)

In ru/mail/chrome/messenger/AccountWizard.dtd:

>  <!-- LOCALIZATION NOTE (usernameExample.label) : do not translate "&quot;jsmith&quot;" in below line -->
> -<!ENTITY usernameExample.label "(например, «i_ivanov»).">
> -<!ENTITY usernameLabel.label "Имя пользователя:">
> +<!ENTITY usernameExample.label "(например, i_ivanov).">
@yshchebet Not sure if you saw this comment, but let's stick with angled quotes.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.